### ISSUE TITLE
Improve the web accessibility

### ISSUE THREAD

**Issue by RVA2869** [govdirectory/website#348]
### The feature

There are different levels of compliance with the WCAG standard. From A to AAA 
[https://www.w3.org/WAI/standards-guidelines/wcag/conformance-logos/#wcag-22](https://www.w3.org/WAI/standards-guidelines/wcag/conformance-logos/#wcag-22)

Some things we could do:
- Add alternative text for images `alt=""` 
- ...

### Why?

This should be the standard procedure. 

### Examples and inspiration

- [enwiki:Web accessibility](https://en.wikipedia.org/wiki/Web_accessibility)
- [ WebAIM WCAG 2 Checklist](https://webaim.org/standards/wcag/checklist)
- WebAIM: [report](https://wave.webaim.org/report#/https://www.govdirectory.org/)
- [https://www.w3.org/WAI/](https://www.w3.org/WAI/)
- [Silktide accessibility checker](https://silktide.com/tools/toolbar/)

**Comment by Abbe98** [comment]
Do you have an example of a non presentation image? I don't think we should have any?

**Comment by Compilearner** [comment]
We can improve the web accessibility on a web page by setting-up the  text-contrast ratio , using alt attribute specially for blind people , using semantic elements, etc.
Give some info where to add this feature.....

**Comment by Ainali** [comment]
> We can improve the web accessibility on a web page by setting-up the text-contrast ratio , using alt attribute specially for blind people , using semantic elements, etc. 

We can only improve it if it is missing somewhere. If you find some place where it does, please point that out specifically so that it can be addressed.

**Comment by RVA2869** [comment]
One thing that comes to mind is the contrast (or lack thereof) of the white text on the orange background.

**Comment by Compilearner** [comment]
Basically , white color on the orange background has poor contrast ratio (3:21 AA) even for normal text, large text and for UI components. On the other hand, black  color has better contrast ratio (6:5 AA) on the orange background for above.
So, It would be more accessible for visitors(low vision) to see black text as compared to white on orange background......

**Comment by Ainali** [comment]
> One thing that comes to mind is the contrast (or lack thereof) of the white text on the orange background.

That's a good point. I think it might be worth exploring a design with darker orange, #d4431d

![image](https://github.com/user-attachments/assets/d53fe733-2185-4acb-958d-f46ef381a2a0)

**Comment by Ainali** [comment]
> There are different levels of compliance with the WCAG standard. From A to AAA

As a meta question to the original post, I think we should decide which level we strive for, and document that in the [CONTRIBUTING](https://github.com/govdirectory/website/blob/main/CONTRIBUTING.md#standards-to-follow) file.

**Comment by Compilearner** [comment]
I have checked the contrast of darker orange (#d4431d) with white color on foreground and its looks more visible even for normal text .
I think it's a good idea to reference the specific color contrast as new issue...

**Comment by Abbe98** [comment]
I would like to close this in favor of more targeted tasks like #411.

In reality different parts of Govdirectory deserves different levels of accessibility efforts, a good example is the country list vs the country map. The country list is at the core of Govdirectory and the accessibility effort is low, the country map is practically gimmick and making it accessible would take a huge effort.

**Comment by Ainali** [comment]
> In reality different parts of Govdirectory deserves different levels of accessibility efforts, a good example is the country list vs the country map. The country list is at the core of Govdirectory and the accessibility effort is low, the country map is practically gimmick and making it accessible would take a huge effort.

I think this is an important point, so important that we should document it somehow.
